### DISCUSSION TITLE
AMD RX6700XT

### DISCUSSION THREAD

**Discussion by dattjc** [Lolliedieb/lolMiner-releases#647] [General]
Updated to Driver 21.6.2 for AMD RX6700XT, unable to mine. Pls help!!!

Build Log: Error: AMD HSA Code Object loading failed.
problem in loading kernel
Active: False (unsupported device or driver version.)

![Screenshot 2021-06-29 145234_LI](https://user-images.githubusercontent.com/83927997/124398389-5ed0fd00-dd0d-11eb-998f-5a022dc6fdd5.jpg)

**Comment by UltimateGregg** [comment]
I'm glad I'm not the only person with this issue. Hopefully, it gets fixed soon.
